src/more-infos/more-info-climate.html

```diff
@@ -50,11 +50,20 @@
         width: 100%;
       }
 
-      paper-range-slider {
-        width: 198px;
-        height: 2px;
-        margin-bottom: 16px;
-        --paper-range-slider-active-color: var(--primary-color);
+      .heat {
+        --paper-slider-active-color: var(--paper-orange-400);
```

Comment:
These colors do not exist in the production build. [Available colors](https://github.com/home-assistant/home-assistant-polymer/blob/master/src/resources/ha-style.html)